libretro-buildbot-recipe.sh

```diff
@@ -930,95 +833,37 @@ while read line; do
 				FORCE_ORIG=$FORCE
 				OLDBUILD=$BUILD
 
-				if [ "${PREVCORE}" = "bsnes" -a "${PREVBUILD}" = "YES" -a "${COMMAND}" = "BSNES" ]; then
-					FORCE="YES"
-					BUILD="YES"
-				fi
-
-				if [ "${PREVCORE}" = "bsnes" -a "${PREVBUILD}" = "YES" -a "${COMMAND}" = "BSNES_JNI" ]; then
-					FORCE="YES"
-					BUILD="YES"
-				fi
-				
-				if [ "${PREVCORE}" = "bsnes_mercury" -a "${PREVBUILD}" = "YES" -a "${COMMAND}" = "BSNES" ]; then
-					FORCE="YES"
-					BUILD="YES"
-				fi
-
-				if [ "${PREVCORE}" = "bsnes_mercury" -a "${PREVBUILD}" = "YES" -a "${COMMAND}" = "BSNES_JNI" ]; then
-					FORCE="YES"
-					BUILD="YES"
-				fi
-
-				if [ "${PREVCORE}" = "gw" -a "${PREVBUILD}" = "YES" -a "${NAME}" = "gw" ]; then
-					FORCE="YES"
-					BUILD="YES"
-				fi
-
-				if [ "${PREVCORE}" = "fuse" -a "${PREVBUILD}" = "YES" -a "${NAME}" = "fuse" ]; then
-					FORCE="YES"
-					BUILD="YES"
-				fi
-
-				if [ "${PREVCORE}" = "81" -a "${PREVBUILD}" = "YES" -a "${NAME}" = "81" ]; then
```

Comment:
hmmm you can't remove these, on android it builds one abi, then the other, then the other, if you don't have this it will only build the first ABI